doc/releases/1.17.0.rst

```diff
@@ -0,0 +1,175 @@
+
+Version 1.17.0
+==============
+
+Installation Changes
+--------------------
+
+- Significant expansion of PypeIt's use of the cache system; see
+  :ref:`data_installation`.  Important changes include that most cached
+  files are now version dependent.  When upgrading to new versions of
+  PypeIt, users should delete their cache and start fresh.
+
+Dependency Changes
+------------------
+
+- Support added for numpy>=2.0.0; numpy<=2.0.0 should still be supported
+- Deprecated support for python 3.10
+- General update to dependencies to be roughly consistent with release
+  of python 3.11
+
+Functionality/Performance Improvements and Additions
+----------------------------------------------------
+
+- Added the ``max_overlap`` parameter, which limits the set of new order
+  traces added, to compensate for orders missed during automated
+  edge-tracing, to those that have less than a given fractional overlap
+  with adjacent orders.
+- Added the ``order_fitrej`` and ``order_outlier`` parameters used to
+  set the sigma-clipping threshold used when fitting Legendre functions
+  to the order widths and gaps.
+- Added the possibility to decide if the extracted standard star
+  spectrum should be used as a crutch for tracing the object in the
+  science frame (before it was done as default).  This is done by
+  setting the parameter ``use_std_trace`` in FindObjPar.
+- Now PypeIt can handle the case where "Standard star trace does not
+  match the number of orders in the echelle data" both in ``run_pypeit``
+  and in ``pypeit_coadd_1dspec``.
+- Added the functionality to use slitless flats to create pixelflats.
+  Note: new frametype ``slitless_pixflat`` is added to the PypeIt
+  frametype list.
+- The created pixelflats are stored in the reduction directory and in
+  the PypeIt cache directory ``data/pixelflats``.
+- Added a functionality that allows, when multiple frames are combined,
+  to scale each frame to have the same mean value before combining. To
+  use this functionality, the new parameter ``scale_mean`` should be set
+  to ``True``.
+- Added the possibility to use the parameter ``fwhm_fromlines`` also for
+  the tilts calibration.
+
+Instrument-specific Updates
+---------------------------
+
+- Improved LRIS frame typing, including the typing of slitless flats and
+  sky flats.
+- Improved HIRES frame typing and configuration setup.
+- Added support for Keck/KCWI BH3 grating configuration.
+- Updated the requirements of a spectral flip for KCWI (blue) data. If
```

Comment:
Should be:

Updated the requirements of a spectral flip for KCWI (blue) data.
If all amplifiers are used, the data will not be flipped in the spectral
direction. Otherwise, the data will be flipped.

Reply:
Thanks!  Fixed.